scss/footer.scss

```diff
@@ -0,0 +1,29 @@
+@import 'colors';
+
+.footer {
+  align-items: center;
+  background-color: $blue;
+  flex-direction: column;
+  display: flex;
+  height: 170px;
+  justify-content: center;
+  margin-top: 100px;
```

Comment:
Instead of defining `margin-top` for the footer, you should define `padding-bottom: 100px` **for the main container**. Practically, you won't notice the difference, but semantically you don't want to define the space between the footer and the main div, but defining how much space you want to leave between the main container bottom border and its content.